app/models/addresslookup/Address.scala

```diff
@@ -60,7 +60,8 @@ case class Address(
   town: Option[String],
   otherLine: Option[String],
   postcode: String,
-  country: Country) {
+  country: Country,
+  subdivision: String) {
```

Comment:
This Address file looks to be copied and pasted from address-reputation-store. The fix should be to just add a dependency on this service but I'm not sure how big of a change it will be. It'll fix a few things in PTA as there is another Country model that looks eerily similar to the one contained in this file.